src/applications/personalization/profile/components/ProfileHeader.jsx

```diff
@@ -124,6 +138,15 @@ const ProfileHeader = ({
                 {getServiceBranchDisplayName(latestBranchOfService)}
               </dd>
             )}
+            {showUpdatedHeader &&
+              totalDisabilityRating && (
+                <dd className="vads-u-margin-top--0p5">
+                  <a className="vads-u-color--white vads-u-font-size--md font-weight--bold">
+                    {totalDisabilityRating}% Service connected{' '}
+                    <i className={`fa fa-angle-double-right`} />
```

Comment:
I'm not gonna lie. I still don't quite get how to use `dd` with or without `dfn` :-/ Honestly our use of `dfn` on line 137 always felt wrong to me (but I could be totally incorrect). What I'd do here is put `<dt className="sr-only">total disability rating</dt>` before this new `dd` that contains the actual disability rating.

Also I think we'll want to add `aria-hidden="true" role="img"` to the `i` element here since it's purely decorative and can be ignored by assistive tech.